apps/ui/src/components/Modal/Account.vue

```diff
@@ -25,7 +25,7 @@ async function handleLogout() {
   emit('close');
 }
 
-watch(open, () => (step = null));
+watch(open, () => (step.value = null));
```

Comment:
Good catch, can we also change `step` to be `const` instead of `let` to avoid it in the future?